battle-engine.js

```diff
@@ -557,24 +549,24 @@ BattlePokemon = (() => {
 		if (this.canMegaEvo && !lockedMove) data.canMegaEvo = true;
 
 		return data;
-	};
-	BattlePokemon.prototype.isLastActive = function () {
+	}
+	get lastActive() {
```

Comment:
This doesn't sound like a boolean-property...
An ``isLastActive`` getter sounds good to me though.